components/ShakeApi.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from "react";
+import { View, Text, ActivityIndicator } from "react-native";
+import styled from "styled-components/native";
+import { Accelerometer } from "expo-sensors";
+import { useFonts, Buda_300Light } from "@expo-google-fonts/buda";
+
+const ActivityText = styled.Text`
+  font-weight: 700;
+`;
+
+const ShakeApi = () => {
+  const [data, setData] = useState({
+    x: 0,
+    y: 0,
+    z: 0,
+  });
+  const [quote, setQuote] = useState({});
+  const [loading, setLoading] = useState(false);
+  const [subscription, setSubscription] = useState(null);
+  const [fontsLoaded] = useFonts({
```

Comment:
Didn't change the font for my own project, so interesting to see how you did it.